sorts/heap_sort.py

```diff
@@ -1,58 +1,56 @@
-"""
-This is a pure Python implementation of the heap sort algorithm.
-
-For doctests run following command:
-python -m doctest -v heap_sort.py
-or
-python3 -m doctest -v heap_sort.py
-
-For manual testing run:
-python heap_sort.py
-"""
+def heapify(arr, root, size):
+    """
+    Heapify a subtree rooted at 'root' to maintain the max-heap property.
 
+    Args:
+        arr (list): The input list.
+        root (int): The index of the root element of the subtree.
+        size (int): The size of the heap.
```

Comment:
As I have typed before `arr` is a weak variable name while `unsorted` is self-documenting variable name.

Please put the datatypes as typehints in the function signature where they can be verified by mypy.

Do not repeat the datatypes here because readers will get confused if one changes but the other does not.
```suggestion
        unsorted: The input list.
        root: The index of the root element of the subtree.
        size: The size of the heap.
```